src/payment/strategies/adyenv2/adyenv2-payment-strategy.ts

```diff
@@ -210,6 +222,12 @@ export default class AdyenV2PaymentStrategy implements PaymentStrategy {
                 throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
             }
 
+            const { on3DSComplete, on3DSLoading } = this._getAdyenV2PaymentInitializeOptions();
+
+            if (!on3DSComplete || !on3DSLoading) {
+                throw new InvalidArgumentError('"options.adyenv2" argument was not provided during initialization.');
```

Comment:
The error message is not accurate. You're throwing an error because the callbacks are not provided. Also, all these really required callbacks? These are marked as optional. So if they are optional, you should just do a null check before using them or provide a fallback function (i.e.: `noop`) before calling them.